yarn-project/pxe/src/pxe_oracle_interface/pxe_oracle_interface.test.ts

```diff
@@ -351,8 +351,9 @@ describe('PXEOracleInterface', () => {
 
       // We set the indexes to WINDOW_HALF_SIZE + 1 so that it's outside the window and for this reason no updates
       // should be triggered.
+      const index = WINDOW_HALF_SIZE + 1;
       await taggingDataProvider.setTaggingSecretsIndexesAsRecipient(
-        secrets.map(secret => new IndexedTaggingSecret(secret, WINDOW_HALF_SIZE + 1)),
+        secrets.map(secret => new IndexedTaggingSecret(secret, index)),
```

Comment:
Introduced this change when I initially bumped the window size to 20 to fix the test. Then I reverted it back to 10 but decided to keep this change around as it improves the test (it makes it robust against window changes).